compiler/rustc_mir_dataflow/src/framework/engine.rs

```diff
@@ -288,7 +288,7 @@ where
             }
 
             None if dump_enabled(tcx, A::NAME, def_id) => {
-                create_dump_file(tcx, ".dot", false, A::NAME, &pass_name.unwrap_or("-----"), body)?
+                create_dump_file(tcx, ".dot", true, A::NAME, &pass_name.unwrap_or("-----"), body)?
```

Comment:
I don't object to this change, but it isn't strictly speaking necessary for this PR, is it?